linux_os/guide/system/software/sap_host/accounts_authorized_local_users/rule.yml

```
@@ -23,6 +23,15 @@ rationale: |-
     requiring access to the system and application accounts for applications not installed
     on the system.
 
+identifiers:
+    cce@sle12: CCE-83195-8
+
+references:
+    disa@sle12: CCI-000366
+    nist@sle12: CM-6(b),CM-6.1(iv)
+    stigid@sle12: SLES-12-010630
+    srg@sle12: SRG-OS-000480-GPOS-00227
+
 severity: medium
 
 references:
```

```suggestion
identifiers:
    cce@sle12: CCE-83195-8

severity: medium

references:
    disa@sle12: CCI-000366
    nist@sle12: CM-6(b),CM-6.1(iv)
    stigid@sle12: SLES-12-010630
    srg@sle12: SRG-OS-000480-GPOS-00227
```

Merge references section

---

actually this files is already present in https://github.com/ComplianceAsCode/content/pull/6631/files#diff-8ee9f0abe56ce47a3a3b0f681f2da65b4713bac4c9537158c8e8f659816ac682 you may remove it from this pull request.